src/dbt/kipptaf/models/students/intermediate/int_students__athletic_eligibility.sql

```diff
@@ -0,0 +1,333 @@
+            and e.grade_level_prev = 8
+            and a.term = 'Q2'
+            and (g.gpa_term >= 2.2 and g.gpa_term <= 2.4)
+            and a.`ada` >= .9
+        then 'Probation - GPA'
+        when
+            a.grade_level = 9
+            and e.grade_level_prev = 8
+            and a.term = 'Q2'
+            and (g.gpa_term >= 2.2 and g.gpa_term <= 2.4)
+            and a.`ada` < .9
+        then 'Probation - GPA and ADA'
+
+        -- semester 1 conditions
+        when
+            a.semester = 'S1'
+            and c.earned_credits_previous_year >= 30
+            and gpa_term >= 2.5
+            and a.`ada` >= .9
+        then 'Eligible'
+        when
+            a.semester = 'S1'
+            and c.earned_credits_previous_year >= 30
+            and g.gpa_term >= 2.5
+            and a.`ada` < .9
+        then 'Probabtion - ADA'
+        when
+            a.semester = 'S1'
+            and c.earned_credits_previous_year >= 30
+            and (g.gpa_term >= 2.2 and g.gpa_term <= 2.4)
+            and a.`ada` >= .9
+        then 'Probation - GPA'
+        when
+            a.semester = 'S1'
+            and c.earned_credits_previous_year >= 30
+            and (g.gpa_term >= 2.2 and g.gpa_term <= 2.4)
+            and a.`ada` < .9
+        then 'Probation - GPA and ADA'
+        when
+            a.semester = 'S1'
+            and c.earned_credits_previous_year < 30
+            and gpa_term >= 2.5
+            and a.`ada` >= .9
+        then 'Ineligible - Credits'
+        when
+            a.semester = 'S1'
+            and c.earned_credits_previous_year >= 30
+            and g.gpa_term < 2.2
+            and a.`ada` >= .9
+        then 'Ineligible - GPA'
+
+        -- semester 2 conditions
+        when
+            a.semester = 'S2'
+            and c.is_cy_credits_on_track
+            and gpa_term >= 2.5
+            and a.`ada` >= .9
+        then 'Eligible'
+        when
+            a.semester = 'S2'
+            and c.is_cy_credits_on_track
+            and g.gpa_term >= 2.5
+            and a.`ada` < .9
+        then 'Probabtion - ADA'
+        when
+            a.semester = 'S2'
+            and c.is_cy_credits_on_track
+            and (g.gpa_term >= 2.2 and g.gpa_term <= 2.4)
+            and a.`ada` >= .9
+        then 'Probation - GPA'
+        when
+            a.semester = 'S2'
+            and not c.is_cy_credits_on_track
+            and (g.gpa_term >= 2.2 and g.gpa_term <= 2.4)
+            and a.`ada` < .9
+        then 'Probation - GPA and ADA'
+        when
+            a.semester = 'S2'
+            and not c.is_cy_credits_on_track
+            and gpa_term >= 2.5
+            and a.`ada` >= .9
+        then 'Ineligible - Credits'
+        when
+            a.semester = 'S2'
+            and c.is_cy_credits_on_track
+            and g.gpa_term < 2.2
+            and a.`ada` >= .9
+        then 'Ineligible - GPA'
+
+        else 'New combination'
+    end as quarter_athletic_eligibility_status,

```

Comment:
via @GabyRangelB 

can we simplify?

https://docs.google.com/document/d/1A76nb5CHlA99LA_1IZYyjUVTvxdyOXMPxmCJYhbF9BQ/edit?tab=t.yvxrkh8ysvce